### DISCUSSION TITLE
Students cannot add .edu address due to new (?) enterprise set-up?

### DISCUSSION THREAD

**Discussion by kljensen** [community/community#137101] [GitHub Education]
### Ahoy,

I teach at Yale. Each year I have about 100 students signing up for GitHub. Those so doing often add their `@yale.edu` email addresses to their accounts in order to receive the Student Developer Pack. _As of August 2024, students cannot add their school addresses to their personal GitHub accounts_! They receive a message saying that the email address is associate with _another_ account. After some digging I discovered that GitHub is rejecting these addresses because there is now an Enterprise Managed User with that email address. Yale has, for a decade at least, had an enterprise GitHub instance, but this has heretofore not been a problem. Furthermore, many hundreds of us here have personal accounts with associated `@yale.edu` addresses. That is all to say: _something has changed._

Alas, I am now stuck in a situation in which my students cannot sign up for the Student Developer Pack. I'm guessing this is a not ideal situation for GitHub because nearly 100% of these students are going into tech firms where they will be in a position to pay large sums for GitHub's [stellar] services.

(You may wonder why one cannot merely use a GitHub Enterprise instance instead of "vanilla" GitHub and my answer is that I make extensive use of GitHub Classroom and GitHub organizations.)

I'd appreciate help with this issue. Absent that, I'd be entertained by other kvetching souls.

-Kyle

**Comment by kljensen** [comment]
My students were able to log in with their enterprise addresses like `Kyle-jensen_yale`, use our centralized auth, and then disassociate their `@yale.edu` address with the enterprise account. Then they could add these to their personal addresses.

**Reply by KerwinFuyihang** [reply]
Hi Kyle, sorry for interrupting you. I am also a student at Yale, and I encountered the same problem. Do you know how I can log in with the 'enterprise address' so that I can try to disassociate my edu email address? Thank you so much.

**Reply by kljensen** [reply]
Kerwin yes I think you just use your CAS credentials

**Comment by patrickfreyer** [comment]
UPDATE: As of Mar 24th '25 GitHub no longer lets you disassociate the primary centralized auth email address, instead you must _unverify_ the email address.
